models/object_detection/model_templates/custom-object-detection/readme.md

```diff
@@ -140,3 +140,39 @@ python eval.py \
    --save-metrics-to ${WORK_DIR}/metrics.yaml \
    --classes ${CLASSES}
 ```
+
+### 11. Optimization
+
+The trained models can be optimized -- compressed by [NNCF](https://github.com/openvinotoolkit/nncf) framework.
+
+At the moment, the following compression methods are supported:
+* int8 quantization
+* sparsity
+* filter pruning
+
+Please, note that NNCF framework requires a dataset for compression, since it makes several steps of fine-tuning after
+compression to restore the quality of the model, so the command line parameters of the compression script are closer
+to the command line parameter of the training script (see the section "7. Training" stated above), but the number of epochs
+required for NNCF compression should not be set by command line parameter, since it is calculated by the script
+`compress.py` itself:
+```
+      python compress.py \
+         --load-weights ${SNAPSHOT} \
+         --train-ann-files ${TRAIN_ANN_FILE} \
+         --train-data-roots ${TRAIN_IMG_ROOT} \
+         --val-ann-files ${VAL_ANN_FILE} \
+         --val-data-roots ${VAL_IMG_ROOT} \
+         --save-checkpoints-to outputs \
+         --nncf-quantization
+```
+
+Note that in the command above the parameter `--nncf-quantization` is used to make int8 quantization of the model.
+You can use the following command line flags to turn on compression methods:
+* `--nncf-quantization` -- int8 compression
+* `--nncf-sparsity` -- sparsity
+* `--nncf-pruning` -- filter pruning
+
+The combinations of several compression methods are also possible, but were not tested thoroughly.
+
+The compressed model can be evaluated and exported to the OpenVINO™ format by the same commands as non-compressed model,
```

Comment:
May be add move optimization step befor eval and export?

Reply:
I think the current flow with Train->Eval->Export->Optimize is ok.